src/frontend/components/TourGuide/TourGuide.scss

```diff
@@ -0,0 +1,77 @@
+.introjs-tooltip {
+  background: #222530;
+  color: var(--text-default);
+  border-radius: var(--space-xs);
+  box-shadow: 0px 4px 20px rgba(0, 0, 0, 0.25);
+  border: 1px solid var(--accent);
+  min-width: 320px;
+}
+
+.introjs-tooltiptext {
+  font-size: var(--text-xl);
+  line-height: 1.5;
+  padding: var(--space-sm);
+}
+
+.introjs-button {
+  border: 2px solid var(--stroke-brand-strong, #ab82fe);
+  border-radius: var(--space-sm, 8px);
+  background: var(--Fill-Brand-strong, #6b2adc);
```

Comment:
Are you sure this variable is being applied, as it can be case sensitive?

Reply:
and this hexacolor?

Reply:
Copy paste from Figma. 
I needed to implement the button since introJS doesnt have a way of passing our buttons from hp-ui.

Reply:
Our variables are all lowercase